vendor/runsvinit/main.go

```diff
@@ -13,52 +13,66 @@ import (
 
 const etcService = "/etc/service"
 
+var (
+	debugf = log.Printf
+	info   = log.Print
+	infof  = log.Printf
+	fatal  = log.Fatal
+	fatalf = log.Fatalf
```

Comment:
given that fatal and info are never overridden, it doesn't seem necessary to alias them here.